### ISSUE TITLE
Remove the option to "Skip sign-in and view limited dashboard" if the admin is the only admin

### ISSUE THREAD

**Issue by eclarke1** [google/site-kit-wp#5380]
## Bug Description

Bug bash issue: https://app.asana.com/0/1202258919887896/1202445528223675 please refer to Asana issue for background

Remove the option to "Skip sign-in and view limited dashboard" if the admin is the only admin, or if there were no modules shared with that admin. If an admin disconnects from Site Kit and only they have shared access to their connected modules (ie. Analytics) remove the option to "Skip sign-in an view limited dashboard". 

## Steps to reproduce

1. Sign in to Site Kit and enable Dashboard Sharing of at least one module with all admins
2. Disconnect Site Kit
3. Go to Site Kit sign in splash page
4. See there is an option to "view shared dashboard" that shouldn't be there.

## Screenshots

[Recording of experience](https://recordit.co/dIC0iH0RNt) - jump to 1 minute 35 seconds

![image.png](https://images.zenhubusercontent.com/5e2ad3803c374569231d8cd7/f0d947fd-be48-4e48-9ea9-4ee5e0289b94)
---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* When there is only one admin on the site, the "View shared Dashboard" link should never appear
* The "View Shared Dashboard" link should never appear when no admins are authenticated with Site Kit
* The "View Shared Dashboard" link should also not appear when no modules are being shared

## Implementation Brief
* Using `assets/js/components/setup/SetupUsingProxyWithSignIn.js`,
	* Update the condition (on top of the existing ones) to display the "Skip sign-in and view limited dashboard" button if:
		* there are multiple admins
			* Query the `core/site` data store via the `hasMultipleAdmins` selector to determine if there are multiple admins
		* there are authenticated admins
			* Query the `core/site` data store via the `hasConnectedAdmins` selector to determine if there are authenticated admins
* Update `assets/js/components/setup/SetupUsingProxyWithSignIn.stories.js` to reflect above changes.

### Test Coverage
* No new tests to be added.

## QA Brief

* Follow the first three steps in the **"Steps to reproduce"** section above.
* Make sure the "Skip sign-in and view limited dashboard" link isn't available.
* In short, if the site only has one admin, the skip sign-in link shouldn't show up.
* For the skip sign-in link to show up, the site needs to have more than one admin and at least one authenticated admin user.

## Changelog entry

* Remove "view limited dashboard" sign-in option when only one admin exists on the site.

**Comment by aaemnnosttv** [comment]
IB ✅

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- II the site only has one admin, the skip sign-in link does not show up. All I can see is the reset link.
- If the site has more than one admin, the skip sign-in link does show up. 

![image](https://user-images.githubusercontent.com/73545194/175029887-ec484557-1eab-40db-910f-a3ec57aa85f6.png)
